.storybook-s2/docs/Release Notes.mdx

```diff
@@ -4,6 +4,42 @@ export default MDXLayout;
 
 # Release Notes
 
+## v0.9.0
+
+### Updates
+
+* [Combobox](?path=/docs/combobox--docs): Support for virtualization and async loading
+* [Dialog](?path=/docs/picker--dialog): Update sizes
+* [Picker](?path=/docs/picker--docs): Support for virtualization and async loading
+* [TableView](?path=/docs/tableview--docs): Support custom menu's in columns
+* Apply `position: relative` to Radio and Checkbox to prevent layout jumps
+
+
+### UNSAFE_className Typescript Error
+
+Style macros passed to `UNSAFE_className` will now result in a TypeScript error. This is not allowed because `UNSAFE_className` is appended to the component’s own styles, not merged. For example, if you pass a style macro class that sets a property but the component also has a class setting the same property, both will be applied and result in undefined behavior depending on the order the CSS was loaded on the page.
+
+We strongly discourage using `UNSAFE_className` because it results in inconsistent UIs and hard to maintain code. Instead, use [React Aria Components](https://react-spectrum.adobe.com/react-aria/index.html) with the [S2 style macro](https://react-spectrum.adobe.com/s2/index.html?path=/docs/style-macro--docs) to create custom components. [Safe style properties](https://react-spectrum.adobe.com/s2/index.html?path=/docs/intro--docs#styling) can be passed to the `styles` prop of an S2 component.
+
+
+### Style macro updates
+
+We have made significant changes to the way our Style Macro generates class names in an effort to make them stable between version. While we work to stablize the style macro class names, we have added a postfix based on the version number so that class names don't conflict with any prior or future version. 
```

Comment:
```suggestion
We have made significant changes to the way our Style Macro generates class names in an effort to make them stable between version. While we work to stabilize the style macro class names, we have added a postfix based on the version number so that class names don't conflict with any prior or future version. 
```